website/docs/d/cloud_vpn_network.html.markdown

```diff
@@ -0,0 +1,53 @@
+---
+subcategory: "Cloud"
+layout: "aci"
+page_title: "ACI: aci_cloud_external_network_vpn_network"
+sidebar_current: "docs-aci-data-source-aci_cloud_external_network_vpn_network"
+description: |-
+  Data source for ACI Template for VPN Network
+---
+
+# aci_cloud_external_network_vpn_network #
+
+Data source for ACI Template for VPN Network
+<b>Note: This resource is supported in Cloud APIC version > 25.0 only.</b>
+
+
+## API Information ##
+
+* `Class` - cloudtemplateVpnNetwork
+* `Distinguished Name` - uni/tn-{tenant_name}/infranetwork-{infra_network_name}/extnetwork-{external_network_name}/vpnnetwork-{vpn_network_name}
+
+## GUI Information ##
+
+* `Location` - Cloud APIC -> Application Management -> External Networks -> VPN Networks
+
+
+
+## Example Usage ##
+
+```hcl
+data "aci_cloud_external_network_vpn_network" "example" {
+  aci_cloud_external_network_dn  = aci_cloud_external_network.example.id
+  name  = "example"
+}
+```
+
+## Argument Reference ##
+
+* `aci_cloud_external_network_dn` - (Required) Distinguished name of parent TemplateforExternalNetwork object.
+* `name` - (Required) Name of the Cloud VPN Network object.
+
+## Attribute Reference ##
+* `id` - Attribute id set to the Dn of the Cloud VPN Network.
+* `remote_site_id` - (Optional) Remote Site ID. 
+* `remote_site_name` - (Optional) Remote Site Name. 
+* `ipsec_tunnel` - (Optional) IPSec tunnel destination. Type: Block.
```

Comment:
should we include the class name of child and cardinality as is done in some other resources/data sources?